fastlane_bot/helpers/routehandler.py

```diff
@@ -1358,14 +1375,13 @@ def _calc_carbon_output(
         Decimal
             The amount out.
         """
+        assert tkn_in not in self.ConfigObj.NATIVE_GAS_TOKEN_ADDRESS,"[routehandler.py _calc_carbon_output] This error is appearing because the function does not expect an input of the native gas token. Please post this error with your run configuration in the Bancor Developers Telegram channel."
```

Comment:
You should probably make the log entry much shorter, as log entries only need to:
1. To include a high-level description of the problem, plus the related state (variables) if relevant
2. To be easily (i.e., uniquely) traceable in the code

You can probably add most of that description as a comment in the code.

Reply:
I shortened the error message.

Reply:
If the stuff that you removed is important, then please add it as a comment above the assertion line.

Reply:
It's not important it was just verbose.